libpkpass/commands/create.py

```diff
@@ -11,23 +11,29 @@ class Create(Command):
     description = 'Create a new password entry and encrypt it for yourself'
     selected_args = Command.selected_args + ['pwname', 'pwstore', 'overwrite', 'stdin', 'keypath',
                                              'nopassphrase', 'nosign', 'card_slot', 'escrow_users',
-                                             'min_escrow', 'noescrow']
+                                             'min_escrow', 'noescrow', 'description', 'authorizer']
 
         ####################################################################
     def _run_command_execution(self):
         """ Run function for class.                                      """
         ####################################################################
-        password1 = getpass.getpass("Enter password to create: ")
-        if password1.strip() == "":
-            raise BlankPasswordError
-        password2 = getpass.getpass("Enter password to create again: ")
-        if password1 != password2:
-            raise PasswordMismatchError
+        if not self.args['description']:
+            self.args['description'] = input("Description: ")
```

Comment:
I like this `description` `authorizer` addition, users should be able to throw this into their rc file and get prompted less.